shared/actions/wallets.tsx

```diff
@@ -641,7 +646,17 @@ const setAccountAsDefault = (_: TypedState, action: WalletsGen.SetAccountAsDefau
   RPCStellarTypes.localSetWalletAccountAsDefaultLocalRpcPromise(
     {accountID: action.payload.accountID},
     Constants.setAccountAsDefaultWaitingKey
-  ).then(() => WalletsGen.createDidSetAccountAsDefault({accountID: action.payload.accountID}))
+  ).then(accountsAfterUpdate =>
+    WalletsGen.createDidSetAccountAsDefault({
+      // @ts-ignore codemod-issue
```

Comment:
What's the TS issue?

Reply:
nice. no issue at all. removed.

Reply:
Maybe the one you copied can be removed too?

Reply:
nope. i tried.

Reply:
@cjb this is the one i copied from. it looks to me like an incorrect type being ignored, and this seems right to me. https://github.com/keybase/client/pull/18460/commits/a27edfd6071e8f06612d6ebc9a356fbb4473ef07